AdminCore.FsmWorkflow/FsmMachines/WorkflowFsmPto.cs

```diff
@@ -0,0 +1,150 @@
+using System.Collections.Generic;
+using AdminCore.Common;
+using AdminCore.Constants.Enums;
+using AdminCore.FsmWorkflow.FsmMachines.FsmLeaveStates;
+using AdminCore.FsmWorkflow.FsmMachines.FsmLeaveTriggers;
+using AdminCore.FsmWorkflow.FsmMachines.FsmWorkflowState;
+using Stateless;
+
+namespace AdminCore.FsmWorkflow.FsmMachines
+{
+    public class WorkflowFsmPto : WorkflowFsm<PtoState, LeaveTriggersPto>
+    {       
+        private StateMachine<PtoState, LeaveTriggersPto>.TriggerWithParameters<EventStatuses, string> LeaveResponseTrigger;
+        
+        public WorkflowFsmPto(WorkflowStateData fsmStateData)
+        {
+            ConfigureFsm(fsmStateData);
+        }
+        
+        public override void ConfigureFsm(WorkflowStateData fsmStateData)
```

Comment:
do you think it would be a good idea to split this function up?

Reply:
Split this into several smaller methods.